src/components/course/course-header/data/hooks/useRedemptionStatus.js

```diff
@@ -17,6 +17,8 @@ const useRedemptionStatus = () => {
 
   const handleRedeemSuccess = (transaction) => {
     queryClient.invalidateQueries({ queryKey: ['policy'] });
+    queryClient.invalidateQueries({ queryKey: ['redeemablePolicies'] });
```

Comment:
[suggestion] Would recommend making use of the query keys map constant variable you created.

Reply:
Updated 👍🏽